sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -728,67 +575,412 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
+      duplicateDetectionHistoryTimeWindow: "PT1M",
+      maxDeliveryCount: 8,
+      enableBatchedOperations: false,
+      autoDeleteOnIdle: "PT1H",
+      authorizationRules: [
+        {
+          claimType: "SharedAccessKey",
+          claimValue: "None",
+          rights: {
+            accessRights: ["Manage", "Send", "Listen"]
+          },
+          keyName: "allClaims_v2",
+          primaryKey: "pNSRzKKm2vfdbCuTXMa9gOMHD66NwCTxJi4KWJX/TDc=",
+          secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
+        },
+        {
+          claimType: "SharedAccessKey",
+          claimValue: "None",
+          rights: {
+            accessRights: ["Manage", "Send", "Listen"]
+          },
+          keyName: "allClaims_v3",
+          primaryKey: "pNSRzKKm2vfdbCuTXMa9gOMHD66NwCTxJi4KWJX/TDc=",
+          secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
+        }
+      ],
+      enablePartitioning: true
+      // maxSizeInMegabytes: 2048,
+      // For partitioned entities, above value is 16384
+      // To be investigated further as part of https://github.com/azure/azure-sdk-for-js/issues/5354
+    },
+    output: {
+      duplicateDetectionHistoryTimeWindow: "PT1M",
+      lockDuration: "PT45S",
+      messageCount: 5,
+      sizeInBytes: 250,
+      defaultMessageTtl: "P2D",
+      deadLetteringOnMessageExpiration: true,
+      enableBatchedOperations: false,
+      maxDeliveryCount: 8,
+      requiresDuplicateDetection: true,
+      requiresSession: true,
+      autoDeleteOnIdle: "PT1H",
+      authorizationRules: [
+        {
+          claimType: "SharedAccessKey",
+          claimValue: "None",
+          rights: {
+            accessRights: ["Manage", "Send", "Listen"]
+          },
+          keyName: "allClaims_v2",
+          primaryKey: "pNSRzKKm2vfdbCuTXMa9gOMHD66NwCTxJi4KWJX/TDc=",
+          secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
+        },
+        {
+          claimType: "SharedAccessKey",
+          claimValue: "None",
+          rights: {
+            accessRights: ["Manage", "Send", "Listen"]
+          },
+          keyName: "allClaims_v3",
+          primaryKey: "pNSRzKKm2vfdbCuTXMa9gOMHD66NwCTxJi4KWJX/TDc=",
+          secondaryKey: "UreXLPWiP6Murmsq2HYiIXs23qAvWa36ZOL3gb9rXLs="
+        }
+      ],
+
+      enablePartitioning: true,
+      maxSizeInMegabytes: 16384,
+      supportOrdering: false,
+
+      forwardDeadLetteredMessagesTo: undefined,
+      forwardTo: undefined,
+      path: undefined,
+      userMetadata: undefined,
+
+      messageCountDetails: undefined,
+      enableExpress: false,
+      entityAvailabilityStatus: "Available",
+      isAnonymousAccessible: false,
+      status: "Active",
+      queueName: alwaysBeExistingQueue
+    }
+  },
+  {
+    testCaseTitle: "pass in forwardDeadLetteredMessagesTo",
+    input: {
+      forwardDeadLetteredMessagesTo:
+        "alwaysBeExistingTopic1/Subscriptions/alwaysBeExistingSubscription1",
```

Comment:
We are using a subscription for the `forwardDeadLetteredMessagesTo` here which is misleading as only queues and topics can be used here.